docs/installation/macos.mdx

```diff
@@ -32,6 +32,14 @@ Select the `GoogleService-Info.plist` file you downloaded, and ensure the "Copy
   caption={false}
 />
 
+## Enabling Outgoing Connections
+
+It is **important** you enable `Outgoing Connections (Client)` on Xcode's `Runner.xcworkspace` as illustrated below:
```

Comment:
as per [this section](https://flutter.dev/desktop#setting-up-entitlements) it is recommended to edit `.entitlements` directly, not via UI. Maybe worth mentioning how to do this manually or just adding a link to this section